queryweaver_sdk/client.py

```
@@ -0,0 +1,295 @@
+    def _graph_name(self, graph_id: str) -> str:
+        """Get the namespaced graph name.
+
+        Args:
+            graph_id: The user-facing graph/database identifier.
+
+        Returns:
+            The namespaced graph name for internal use.
+        """
+        graph_id = graph_id.strip()[:200]
+        if not graph_id:
+            raise ValueError("Invalid graph_id, must be non-empty and less than 200 characters.")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)